### DISCUSSION TITLE
谁给解读下，这是不是政治搏斗

### DISCUSSION THREAD

**Discussion by zongjiangU** [WeNeedHome/SummaryOfLoanSuspension#681] [Ideas]
+ 上海顶着天大压力封城
+ 河南村镇银行跑路
+ 北京公共场所疫苗政策一天一改
+ 老总视察武汉指导宁损经济严守病毒，与国务院取消行程码星相隔五分钟
... ...

**Comment by JACWATA** [comment]
别问，问就是加速

**Comment by C-Joey** [comment]
不做中南海听床师
任何的行为都是博弈的结果

**Comment by oceanzhang860411** [comment]
覺得是搏鬥你別來啊，又沒人求著你

**Reply by fnnnnng** [reply]
好奴才

**Comment by yuntun7zz** [comment]
这跟任何政治都无关，我们老百姓只想要回属于自己的房子，也不想关心任何政治

**Reply by ghost** [reply]
好啊，不想关心就别想拿回来

**Reply by Joseph-Wheat-Carryier** [reply]
觉悟不够啊，在这种政体国家下面想独善其身是不可能的

**Reply by redapple2021** [reply]
你不关心，那就别到墙外来呗，你不知道这堵墙也是政治吗？还有房子也别要了继续还贷吧

**Reply by MrThanlon** [reply]
> 你不关心，那就别到墙外来呗，你不知道这堵墙也是政治吗？还有房子也别要了继续还贷吧

我寻思着github什么时候是墙外了？再说墙什么时候是政治了，你说是就是？

**Reply by tanuk1997** [reply]
> 这跟任何政治都无关，我们老百姓只想要回属于自己的房子，也不想关心任何政治

那就祝你继续烂尾，哈哈哈，煞笔只想要别人牺牲为你们争取利益，滚你🐎

**Comment by wedxz** [comment]
> 这跟任何政治都无关，我们老百姓只想要回属于自己的房子，也不想关心任何政治

觉悟真高…

**Reply by volans19** [reply]
党最喜欢这种人了哈哈，最好控制最好分化的就是这样的，一拿一个准，话说回来，烂尾房也就只能坑这种人了，别的还真没人选

**Comment by AntEffect** [comment]
社会经济全民政治化又不是没有过，你看下现在这位的套路是跟谁学的不就明白了？

**Comment by blue-ling233** [comment]
屁的搏斗，若都依法治国，哪有这些屁事

**Comment by lance111l** [comment]
疫情复杂  应对措施摸索中 可以理解 
你有啥好方法吗
